src/test/java/com/uwetrottmann/tmdb2/annotations/RequiresGuestSession.java

```diff
@@ -0,0 +1,11 @@
+package com.uwetrottmann.tmdb2.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface RequiresGuestSession {
```

Comment:
Added RequireGuestSession Annotation to determine if a test requires Guest Session in order to initialize Guest Data